src/cijoe/core/scripts/example_script_default.py

```diff
@@ -42,21 +42,21 @@
 def add_args(parser: ArgumentParser):
     """Optional function for defining command-line arguments for this script"""
     parser.add_argument(
-        "--message", type=str, default=None, help="The message to be printed"
+        "--repeat",
+        type=int,
+        default=1,
+        help="Amount of times the message will be repeated",
     )
 
 
-def main(args: Namespace, cijoe: Cijoe, step: dict):
+def main(args: Namespace, cijoe: Cijoe):
     """Entry-point of the cijoe-script"""
 
-    # Grab message from command-line arguments, if none is given, grab it from
-    # the configuration-file
-    message = getattr(args, "message", None) or cijoe.getconf(
-        "example.message", "Hello World!"
-    )
+    # Grab message from the configuration-file
+    message = cijoe.getconf("example.message", "Hello World!")
```

Comment:
Why drop the message from cli-arg / workflow-arg?

Reply:
For simplificity's sake, I've changed the `message` variable back to only being defined in the config file. In the last PR, I added it as cli-arg to give an example of how to do this, but I think it's cleaner to just use `repeat` for this cause. This way there are two variables, `message` and `repeat`, and one is an example of how to get config-variables, and the other is an example of how to get cli-arguments. :)